graph-proxy/argo-workflows-openapi/build.rs

```diff
@@ -12,7 +13,23 @@ fn main() {
         .unwrap()
         .text()
         .unwrap();
-    let schema = serde_json::from_str::<schemars::schema::RootSchema>(&raw_schema).unwrap();
+    let mut schema: RootSchema = serde_json::from_str(&raw_schema).unwrap();
+
+    if let Some(Schema::Object(ref mut pod_gc)) = schema
```

Comment:
Could we add a comment explaining why this is done with a link to the upstream issue?